wallet/src/account.rs

```diff
@@ -0,0 +1,51 @@
+use crate::types;
+
+/// Result type for accounts-related operations that can fail.
+pub type Result<T> = std::result::Result<T, failure::Error>;
+
+/// Root KeyPath used for wallet accounts.
+#[inline]
+pub fn account_keypath(index: u32) -> types::KeyPath {
+    types::KeyPath::default()
+        .hardened(3)
+        .hardened(4919)
+        .hardened(index)
```

Comment:
Why 3 and 4919?

Reply:
It was defined here: https://github.com/witnet/sheikah/blob/master/app/main/api/handlers/generateAddress.ts#L110